cpp/oneapi/dal/table/test/csr_accessor.cpp

```diff
@@ -0,0 +1,185 @@
+/*******************************************************************************
+* Copyright 2021 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#include "oneapi/dal/table/csr.hpp"
+#include "oneapi/dal/table/csr_accessor.hpp"
+#include "oneapi/dal/test/engine/common.hpp"
+
+namespace oneapi::dal {
+
+TEST("can read csr table via csr accessor") {
+    using oneapi::dal::detail::empty_delete;
+
+    double data[] = { 1.0, 2.0, 3.0, 4.0, 1.0, 11.0, 8.0 };
```

Comment:
What about using templates to test different versions of this functionality?

Reply:
Done